server/issue.go

```diff
@@ -728,6 +728,20 @@ func (p *Plugin) getIssueDataForCloudWebhook(instance Instance, issueKey string)
 	return issue, nil
 }
 
+func (p *Plugin) getEpicName(epicID string, instanceID types.ID) (string, error) {
+	instance, err := p.instanceStore.LoadInstance(instanceID)
+	if err != nil {
+		return "", err
+	}
+
+	epic, err := p.getIssueDataForCloudWebhook(instance, epicID)
+	if err != nil {
+		return "", err
+	}
```

Comment:
Note that this will only work for Jira Cloud instances. In order to support this feature for Jira Server instances, we would need to instantiate a Jira client for the user that caused the webhook event. I think this epic name feature doesn't warrant introducing that complexity. Nothing to change here in that regard 🙂

One suggestion to log the epic fetch error here:

```suggestion
	_, ok := instance.(*cloudInstance)
	if !ok {
		return "", errors.Errorf("Must be a JIRA Cloud instance, is %s", instance.Common().Type)
	}

	epic, err := p.getIssueDataForCloudWebhook(instance, epicID)
	if err != nil {
		p.API.LogError("Error fetching epic name for cloud webhook", "err", err.Error())
		return "", err
	}
```